tests/filecheck/dialects/func/func_ops.mlir

```diff
@@ -1,40 +1,55 @@
-// RUN: xdsl-opt %s | xdsl-opt --print-op-generic | filecheck %s
+// RUN: xdsl-opt --print-op-generic %s | xdsl-opt | filecheck %s
 
-"builtin.module"() ({
-  "func.func"() ({
+builtin.module {
+
+  func.func @noarg_void() {
     "func.return"() : () -> ()
-  }) {"sym_name" = "noarg_void", "function_type" = () -> (), "sym_visibility" = "private"} : () -> ()
+  }
 
-   // CHECK:      "func.func"() ({
+   // CHECK:      func.func @noarg_void() {
    // CHECK-NEXT:   "func.return"() : () -> ()
-   // CHECK-NEXT: }) {"sym_name" = "noarg_void", "function_type" = () -> (), "sym_visibility" = "private"} : () -> ()
+   // CHECK-NEXT: }
 
-  "func.func"() ({
+  func.func @call_void() {
     "func.call"() {"callee" = @call_void} : () -> ()
     "func.return"() : () -> ()
-  }) {"sym_name" = "call_void", "function_type" = () -> (), "sym_visibility" = "private"} : () -> ()
+  }
 
-   // CHECK: "func.func"() ({
+   // CHECK-NEXT: func.func @call_void() {
```

Comment:
I would leave this as `CHECK`. That makes it easily reorderable. I would even say that if we move to something based on split input for the invalid tests, we could consider doing the same for the valid once, so making all of these individual tests what a seperator in between. No strong pref here though.